geoh5py/ui_json/ui_json.py

```diff
@@ -220,6 +220,8 @@ def validate_data(
 
         ui_json = self.model_dump(exclude_unset=True)
         for field in self.model_fields_set:
+            if self.is_disabled(field):

```

Comment:
does is disable is also checking for :
group dependencies? object dependencies? 
If so I'm ok! :-)